<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="Dayanands-MacBook-Air.local" failures="2" tests="6" name="Test" time="41.702" errors="0" timestamp="2025-04-07T21:35:25 IST">
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="8.568"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="5.99"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="5.567"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="3.666"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="0.628">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:51)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="0.895">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:66)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
</testsuite> <!-- Test -->
